src/app/services/notification.service.ts

```diff
@@ -76,11 +76,10 @@ export class NotificationService {
         const identifier = this.getNewIdentifier();
 
         this.pendingActions = [
-            {
-                identifier,
-                title: message.title,
-                description: message.description
-            },
+            Object.assign({}, message, {
```

Comment:
_nitpicky comment_: Is there any specific reason why you have to use `Object.assign` instead of simply destructuring message? `{ ...message, identifier }`. They are supposed to do the same if I am not mistaken.

Reply:
I never thought of that. Thank you!